lib/openstax/aws/packer_1_2_5.rb

```diff
@@ -53,17 +53,13 @@ def run
       @logger.info("**** DRY RUN ****") if dry_run
       @logger.info("Running: #{command}")
 
-      if !dry_run
```

Comment:
I think the line 53 needs to be updated from `dry_run` to `@dry_run` as well

Reply:
I think they should be the other way around, right? When i try running the script i get errors if they don't have the `@` which seems logical but i don't know why were they without them in the first place

Reply:
when I added the new file for Packer 1.4, I probably added this dry run code but never tested it 😬

Reply:
fixed, pls someone approve this so it can finally get out of code @jpslav @janKozie1